src/main/java/org/jabref/gui/FileDialog.java

```diff
@@ -114,6 +115,14 @@ public void setDefaultExtension(FileExtensions extension) {
     }
 
     /**
+     * Sets the initial file name, useful for saving dialogs
+     * @param fileName
+     */
+    public void setInitialFileName(String fileName) {
```

Comment:
Instead of extending a deprecated class, can the FileDialog below be replaced by the new DialogService?